src/main/webapp/app/shared/metis/posting-create-edit-modal/answer-post-create-edit-modal/answer-post-create-edit-modal.component.ts

```diff
@@ -42,7 +42,7 @@ export class AnswerPostCreateEditModalComponent extends PostingCreateEditModalDi
     resetFormGroup(): void {
         this.formGroup = this.formBuilder.group({
             // the pattern ensures that the content must include at least one non-whitespace character
-            content: [this.posting.content, [Validators.required, Validators.maxLength(this.maxContentLength), Validators.pattern(/^(\n|.)*\S+(\n|.)*$/)]],
+            content: [this.posting.content, [Validators.required, Validators.maxLength(this.maxContentLength), Validators.pattern(/^(\n|\r|.)*\S+(\n|\r|.)*$/)]],
```

Comment:
On Windows, both \r\n get used. 
```suggestion
            content: [this.posting.content, [Validators.required, Validators.maxLength(this.maxContentLength), Validators.pattern(/^(\r\n|\n|\r|.)*\S+(\r\n|\n|\r|.)*$/)]],
```
Since the comment talks about "whitespace" instead of newlines, I think also this would work (see https://www.w3schools.com/jsref/jsref_regexp_whitespace.asp)
```suggestion
            content: [this.posting.content, [Validators.required, Validators.maxLength(this.maxContentLength), Validators.pattern(/^(\s|.)*\S+(\s|.)*$/)]],
```
It would also be great if you could find a way to remove the duplicated regex. In fact, this whole method looks duplicated and should probably be changed.

Reply:
> On Windows, both \r\n get used.
> 
> Since the comment talks about "whitespace" instead of newlines, I think also this would work (see https://www.w3schools.com/jsref/jsref_regexp_whitespace.asp)
> 
> It would also be great if you could find a way to remove the duplicated regex. In fact, this whole method looks duplicated and should probably be changed.
> 
> Edit: I did not see the Star at first glance, so \r\n is actually already matched. But still, I think \s would be a cleaner option.

I'm working on Windows and currently with the new REGEX pattern, the save button is activated as intended. I will not use '\s' because we want to keep the button deactivated when whitespace is entered into the post body.

Reply:
Duplication removed